docs/sftt/sftt-api-spec.md

```diff
@@ -1147,13 +1174,117 @@ Reservations imported successfully.
 
 If the request was malformed.
 
-##### `401 Unauthorized`
+##### `401 UNAUTHORIZED`
 
 If the user is not a super admin.
 
-### Import Trees
+### Import Sites
+
+`POST api/v1/protected/import/sites`
+
+Used to import sites into the database. See below for a description of each ambiguous field.
+
+`status`: Alive or Dead but standing<br/>
```

Comment:
You should add "quotes" around example values, is it "Dead but standing"?

Reply:
Done, also for other values